src/plugins/data/public/search/search_interceptor/search_interceptor.ts

```diff
@@ -266,10 +277,27 @@ export class SearchInterceptor {
         })
       : undefined;
 
+    // track if this search's session will be send to background
+    // if yes, then we don't need to cancel this search when it is aborted
+    let isSavedToBackground = false;
```

Comment:
While unskipping the remaining tests, I realized that this older implementation from main had some benefits. So I decided to get it back and unskip related tests